Neos.Neos/Classes/Command/CrCommandController.php

```diff
@@ -76,7 +76,7 @@ public function exportCommand(string $path, string $contentRepository = 'default
     /**
      * Import the events from the path into the specified content repository
      *
-     * @param string $path The path for storing the result
+     * @param string $path The path of the stored events like resource://Neos.Demo/Private/Content
```

Comment:
@mhsdesign But this is not correct. The path is enough like `Packages/Sites/Neos.Demo/Private/Content`

Reply:
hmm i prefer the abstraction via the resource manager as this makes it a detail where the package is stored inside `Packages`

And i tested it - it works ^^